# Remote Management Request
temoto_2/RMPRequest rmp

# The name of the robot to load 
string robot_name

---

# Remote Management Response
temoto_2/RMPResponse rmp
